include/katanoisi/rest_server.h

```diff
@@ -49,7 +49,7 @@ class rest_server {
                             Http::ResponseWriter response);
 
   std::vector<std::pair<fasttext::real, std::string>>
-  askClassification(std::string &text, std::string &domain, int count);
+  askClassification(std::string &text, std::string &domain, int count, float threshold);
```

Comment:
Why not passed by reference?

Reply:
Changed in next commit

Reply:
If passed by reference, we can't set a default right value, except if const which is not the case in fastText.